apparmor.d/abstractions/bus/org.freedesktop.PackageKit

```diff
@@ -2,6 +2,41 @@
 # Copyright (C) 2023 Alexandre Pujol <alexandre@pujol.io>
 # SPDX-License-Identifier: GPL-2.0-only
 
+  dbus receive bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Transaction
+       member=GetUpdates
+       peer=(name=:*),
+
+  dbus receive bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.DBus.Properties
+       member=GetUpdates
+       peer=(name=:*),
+
+  dbus receive bus=system path=/org/freedesktop/PackageKit
+       interface=org.freedesktop.DBus.Introspect
+       member=Introspect
+       peer=(name=:*),
+
+  dbus send bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Transaction
+       member={Destroy,Finished,Package}
+       peer=(name=:*),
```

Comment:
Remove this. It gives to much access use a dbus directive in the profile that needs it.

See: https://apparmor.pujol.io/development/dbus/#dbus-abstractions